MANIFEST.in

```diff
@@ -1 +1,2 @@
-recursive-include wof/flask/templates *
\ No newline at end of file
+recursive-include wof/flask/templates *
+recursive-include wof/examples/flask/odm2/timeseries *
```

Comment:
By moving the examples directory inside the module we package those files inside the `wof` namspace at `site-package`. The previous configuration would require a module named `examples`, which is odd and would bring confusion. IMO those below inside the main module.